cmd/psa/verify_as_attester.go

```diff
@@ -107,6 +111,14 @@ to be created on the fly based on the attester's claims and signing IAK.
 		"nonce-size", "n", 48, "nonce size (32, 48 or 64)",
 	)
 
+	attesterIsInsecure = cmd.Flags().BoolP(
```

Comment:
it might be helpful, if the README.md can be elevated to express what the default behaviour is and also availability & how to use the supplied arguments

Reply:
I've aded a note to README-PSA.md